### DISCUSSION TITLE
Backend / UI for posting

### DISCUSSION THREAD

**Discussion by vahidtakro** [timlrx/tailwind-nextjs-starter-blog#236] [Ideas]
I was thinking how to make it possible to post with this template deployed on vercel, with mobile phone, not just microsoft visual studio or any other code editors.
like how Medium works, how to write a new post within the app itself, or even a backend.
I searched for a backend, I found out some headless CMS like Strapi, which creates two directories, one for backend and one for frontend, and it fetch the data from the backend, which are built in a data.json file.
so my question is, what do you recommend me to use to achieve the goal I mentioned?
if your idea is using a headless CMS , which one do you recommend? and how to connect it with mdx files? so it requires to create a mdx file on new posts...

Thanks.

**Comment by timlrx** [comment]
My recommendation would probably be a git based CMS like forestry.io or netlify CMS. Not sure how mobile friendly they are though. Probably forestry.io would be a better fit as they have a cloud hosted solution that connects with your repository.

If you want to go with the Strapi, contentful, sanity.io model, it might be better to use their templates rather than this markdown based template.
